controls/SV-204409.rb

```diff
@@ -13,10 +13,10 @@
     # grep dcredit /etc/security/pwquality.conf
     dcredit = -1
```

Comment:
@ssayed118 this value needs to be parameterized. Also, please look at the previous controls, the default value in inspec.yml is positive and you just add the `-` sign where needed.
This also needs to be corrected for the next control

Reply:
commit link: https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/commit/0ab343207b80d67c634ca48116b52c58749ad762

Reply:
`=` was missing after `cmp`. Added it